data/test/nwp_single_observation.zarr/.zattrs

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
i would better tempted to have a pytest fxiture that makes a ramdom xarray file. You can then save it somewhere temporary. I think this is neater than commiting the test data file. Its also easier to update then

Reply:
How would I then compare the output of the processing modules if the input data is random?

Reply:
could use a seed, perhaps leave it for the moment, and we can change it later